<ui version="4.0" >
 <class>Plot_Options_Dialog</class>
 <widget class="QDialog" name="Plot_Options_Dialog" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>358</height>
   </rect>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Plot Options</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="gen_options_tab" >
      <attribute name="title" >
       <string>General Options</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget" >
       <property name="geometry" >
        <rect>
         <x>33</x>
         <y>50</y>
         <width>295</width>
         <height>124</height>
        </rect>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="1" >
         <widget class="QLabel" name="label_3" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text" >
           <string>X Axis</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="label_4" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text" >
           <string>Y Axis</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="min_label" >
          <property name="text" >
           <string>Minimum:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="xmin_lineEdit" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QLineEdit" name="ymin_lineEdit" />
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="max_label" >
          <property name="text" >
           <string>Maximum:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLineEdit" name="xmax_lineEdit" />
        </item>
        <item row="2" column="2" >
         <widget class="QLineEdit" name="ymax_lineEdit" />
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="axis_text_label" >
          <property name="text" >
           <string>Label:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLineEdit" name="xlabel_lineEdit" >
          <property name="text" >
           <string>x-axis</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" >
         <widget class="QLineEdit" name="ylabel_lineEdit" >
          <property name="text" >
           <string>y-axis</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QCheckBox" name="logy_cb" >
       <property name="geometry" >
        <rect>
         <x>43</x>
         <y>220</y>
         <width>83</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text" >
        <string>Log Y Axis</string>
       </property>
      </widget>
      <widget class="QLabel" name="plot_title_label" >
       <property name="geometry" >
        <rect>
         <x>33</x>
         <y>10</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text" >
        <string>Plot Title:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QCheckBox" name="logx_cb" >
       <property name="geometry" >
        <rect>
         <x>43</x>
         <y>190</y>
         <width>83</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text" >
        <string>Log X Axis</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="plottitle_lineEdit" >
       <property name="geometry" >
        <rect>
         <x>100</x>
         <y>10</y>
         <width>291</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text" >
        <string>Plot Title</string>
       </property>
      </widget>
      <widget class="QPushButton" name="toggleGrid_btn" >
       <property name="geometry" >
        <rect>
         <x>150</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text" >
        <string>Toggle Grids</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="trace_options_tab" >
      <attribute name="title" >
       <string>Trace Options</string>
      </attribute>
      <layout class="QGridLayout" >
       <item rowspan="4" row="0" column="0" >
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_11" >
           <property name="text" >
            <string>Active Plot Traces</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QListWidget" name="activePlotListWidget" />
         </item>
        </layout>
       </item>
       <item row="0" column="1" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Line Style:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QComboBox" name="lstyle_comboBox" />
       </item>
       <item row="1" column="1" >
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>Line Width:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Marker Style:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="QComboBox" name="mstyle_comboBox" />
       </item>
       <item row="3" column="1" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Marker Size:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QDoubleSpinBox" name="lw_spinBox" >
         <property name="minimum" >
          <double>0.250000000000000</double>
         </property>
         <property name="maximum" >
          <double>50.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.250000000000000</double>
         </property>
         <property name="value" >
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="2" >
        <widget class="QDoubleSpinBox" name="ms_spinBox" >
         <property name="minimum" >
          <double>0.250000000000000</double>
         </property>
         <property name="maximum" >
          <double>50.000000000000000</double>
         </property>
         <property name="singleStep" >
          <double>0.250000000000000</double>
         </property>
         <property name="value" >
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="ColorButton" name="lineColorBtn" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>Line Color...</string>
         </property>
         <property name="toolButtonStyle" >
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="2" >
        <widget class="ColorButton" name="markerColorBtn" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>Marker Color...</string>
         </property>
         <property name="toolButtonStyle" >
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Plot_Options_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Plot_Options_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
